WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -466,8 +464,8 @@ - (void)configureBlogCell:(WPBlogTableViewCell *)cell atIndexPath:(NSIndexPath *
     }
     cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
     cell.selectionStyle = self.tableView.isEditing ? UITableViewCellSelectionStyleNone : UITableViewCellSelectionStyleBlue;
-    cell.imageView.layer.borderColor = [UIColor whiteColor].CGColor;
-    cell.imageView.layer.borderWidth = 1.5;
+    cell.imageView.layer.borderColor = self.tableView.separatorColor.CGColor;
```

Comment:
Thanks @mattmiklic. I'm just posting another PR to switch the borders to white (so you'll only see them on selection), which matches the blog selector in Calypso (that also has a white background).